.github/workflows/pull-evaluation-tests.yaml

```diff
@@ -0,0 +1,210 @@
+name: "Evaluation tests"
+run-name: "Evaluation tests"
+
+# Required repository secrets:
+# - AICORE_SECRET
+# - COMPANION_CONFIGMAP
+# - GARDENER_TEST_CLUSTER_URL
+# - GARDENER_TEST_CLUSTER_AUTH_TOKEN
+# - GARDENER_TEST_CLUSTER_CA_DATA
+
+on:
+  pull_request_target:
+    branches:
+      - main
+    paths:
+      - "src/**"
```

Comment:
I think, the evaluation must run if its logic changes. Should we include evaluation related paths to the GH PR event?

Reply:
Its `pull_request_target` so the evaluation tests will run from main branch. So if changes are made to evaluation tests logic we have to test it after the PR is merged.

Reply:
Is it not better to run evaluation test scripts to check if scripts have issues?
We run test pipelines if we change a unit test for example.

Reply:
updated